addons/GGS/src/components/keybind/ggs_keybind_gp.gd

```diff
@@ -40,24 +38,24 @@ func use_axis_icon(value: int, axis: int) -> void:
 		match axis:
```

Comment:
The previous frame numbers were correct. I just couldn't find the sprites that I used for the glyphs so I had to leave the `AnimatedTexture` that's attached to the ggsKeybindGp as is. Which I assume caused an error.
Godot supports up to 22 gamepad buttons, so that's why I thought frames 23-30 can be used to display an icon for the analog stick directions (e.g. frame `25` is the glyph icon for the "Left Stick UP" or `30` is for "Right Stick DOWN").
So I guess the correct structure for the icon resource file should be an animated texture with 30 frames. Frames that don't have a sprite or are not used should just have a blank/transparent image as a placeholder. I think that'd fix the issue.

Note that frames 12-15 are used for d-pad buttons, which are different than the analog stick directions.

Reply:
Gotcha. Fix pushed. I've expanded the frames on gamepad.tres to have 31 frames. The 31st is for transparent.png that we'll set to for out-of-bounds frames.